### ISSUE TITLE
twitter: normalize /status/ and /statuses/ in tweet URLs

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy#618]
twitter supports both? but doesn't redirect either?! ugh. news to me. [from IRC](http://indiewebcamp.com/irc/2016-02-16#t1455641882655):

```
aaronpk hm yes now even though i constructed the comment url with both tweet IDs, it still isn't finding my original post
aaronpk https://brid.gy/comment/twitter/aaronpk/686998035889455104/699605447414804480
...
snarfed aaronpk: oh man. your twitter u-syndication on https://aaronparecki.com/2016/01/12/14/webmention is .../statuses/..., which works, but i think twitter generally uses .../status/..., and bridgy doesn't (yet) know to normalize those
aaronpk snarfed: oh! umm
aaronpk but it was finding replies before...?
snarfed aaronpk: you've always used /statuses/ ?
aaronpk one way I was syndicating in my old site used that path yes
```

i guess twitter needs its own [`Facebook.canonicalize_syndication_url`](https://github.com/snarfed/bridgy/blob/3089e3d9d73dc9afe9e3317ad3bcf0dabb309333/facebook.py#L152) now. :(

**Comment by snarfed** [comment]
cc @aaronpk

**Comment by snarfed** [comment]
fixed by #634
